accounts/abi/bind/util_test.go

```diff
@@ -53,6 +53,7 @@ var waitDeployedTests = map[string]struct {
 	},
 }
 
+//nolint:staticcheck
```

Comment:
I don't like this. staticcheck warnings are pretty much always fixable. It's one of these tools where false positives are rare. So whatever it is reporting here, please fix it.